config/workload_all.xml

```diff
@@ -33,7 +33,7 @@
     <useThinkTime>true</useThinkTime>
     <works>
         <work>
-          <time>30</time>
+          <time>300</time>

```

Comment:
Hey @psudheer21 - this changes the think time from 30 to 300, though the title is only about log message removal. Is this intentional? If so, could we change the title and also add a comment on top referencing the section of the spec, etc just as with the number of terminals above?

Reply:
It is intentional.
There is no time factor in the spec. 
ITs just that we need to run it sufficiently long to get proper results..

Reply:
Got it. Would be great to add a comment on top saying we're running the workload for 300 seconds by default.